src/components/SecretsModal/Annotations.js

```diff
@@ -62,7 +62,7 @@ const Annotations = props => {
         <Add className="addIcon" onClick={handleAdd} />
       </div>
       {invalidFields.find(field => field.includes('annotation-value')) !==
-        undefined && <p className="invalidAnnotation">Required.</p>}
+        undefined && <p className="invalidAnnotation">Server URL Required.</p>}
```

Comment:
I'd rather the case be like that of a normal sentence (this is from the create pipelinerun page) to be consistent, e.g.: `Pipeline cannot be empty` so let's go with "Server URL required"